### PR TITLE
fastBPE recipe

### PR DESCRIPTION
**Pull request by deepali-c**

### CODE REVIEW

**Comment by jayfurmanek** [comment]
@deepali-c Is this ready for review?

File: conda-recipes/fastBPE-feedstock/.ci_support/linux_python3.6.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+c_compiler:
+- gcc
+c_compiler_version:
+- '8'
+channel_sources:
+- conda-forge,defaults
```
Comment (jayfurmanek) [inline conda-recipes/fastBPE-feedstock/.ci_support/linux_python3.6.yaml]: probably don't need conda-forge on the sources here.

**Comment by hartb** [comment]
Interesting! Don't follow all of this, but looks OK to my untrained eye. Will defer to the others.

Thank you, @deepali-c!

File: conda-recipes/fastBPE-feedstock/.ci_support/linux_python3.6.yaml (hunk 1 above)
Comment (deepali-c) [reply conda-recipes/fastBPE-feedstock/.ci_support/linux_python3.6.yaml]: sure, updating it now.

**Comment by deepali-c** [comment]
@jayfurmanek, I have incorporated the review comment and made minor changes. Requesting you to please review once.

**Comment by deepali-c** [comment]
@jayfurmanek, please let me know if I can merge this one.

File: .travis.yml [hunk 2]
```diff
@@ -23,17 +23,22 @@ env:
     - secure: "JZRf0nvjA0bNtlHCo2FiF9Xls11cYLoEK4JComWELDq0Qi1wOtlORR9t9RgyOCeldUt0n8ZiDtGVh42jCyI9xqo+lMpzZsp4GWu3apMSiDfFRNhDrakYxU1FMefathMHBCTk8rmq+P1LX7i/V+wPbNyzoSoNzcci3FmEc6eRUEGM/sBgHqH/xyvRFGZnSXbOMTigzgVSruZOVtL6opxHfo1RnxlU/dR5lytKIKm00r+GPK1w9tuwOPTdGpZUaWA6/aoxMQkUNnKqrZq382xBJ6TFgzmkkY80wJFnvchg0NuPqI7oNoA8z30W0vgYbZ7nno3SExjoCYbwCLWPlwIwsCEipEUG1lFPvVwAMELkCWu22JX95e34ZKxd5SR143/509CFMKHK5UrvLqg4tI7EgjrRpLcEI8utvNdjTCp/KUOcXbLq+cw1TMnzDwFuIdgPfsKGrTNreuaiHxCdt2NKuy8oD497WcMOUpq4rWoWXjDXeUfwxOt6GEH1F07Em5Ng0mSiwyCHxiL5XERpYj4lAnSnMBIaIXaHOWlGpron0jAtcpTu3lIY2rF7zdpUFSGNVnaiwwHUD7RDxy96zc/ig0X7DVIofeNTTML/rYsHEbB+hO92sBll+lxEq1xjeAO8Khsp0ohJLOyM3iglBgHxXrSfLhcYk6pmKrOqi1onJK0="
 
 
-jobs:
+matrix:
+#py36 and py37 builds for ppc64le and x64
   include:
+    - env: CONFIG=linux_ppc64le_python3.6 UPLOAD_PACKAGES=False DOCKER_IMAGE=condaforge/linux-anvil-ppc64le
```
Comment (jayfurmanek) [inline .travis.yml]: Do you want UPLOAD_PACKAGES as `False`? We want this package out in the Supplementary Channel, correct?
Maybe change that to `True`?

Comment (deepali-c) [reply .travis.yml]: Yes, correct I'll update it.

**Comment by deepali-c** [comment]
To get it merge easily, I have squashed some of the older commits.

**Comment by deepali-c** [comment]
@jayfurmanek, please suggest if this can be merged.

**Comment by jayfurmanek** [comment]
Looks good now! Thanks!

**Comment by deepali-c** [comment]
Thanks @jayfurmanek , merging it now.

**Pull request merged**